server/src/main/java/fr/uga/l3miage/tp1/exo2/models/NetflixAccountEntity.java

```diff
@@ -0,0 +1,30 @@
+package fr.uga.l3miage.tp1.exo2.models;
+
+import jdk.jfr.Enabled;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "netflix_account_entity")
+public class NetflixAccountEntity {
+
+    @Id
+    @Column(
+            name = "id"
+    )
+    private Long id;
+
+    @Column(
+            name = "nb_devices"
+    )
+    private int nb_devices;
+
+    @Column(
+            name = "typeAccount"
+    )
+    private String typeAccount;
+
+    @OneToOne
+    private NetflixUserEntity uuid_user;
```

Comment:
Encore une erreur de ma part, mais lorsque vous êtes en mode unidirectionnel et que la clé étrangère est un champ spécifique, il faut utiliser l'annotation `@JoinColumn(name="uuid_user")`
```suggestion
    @OneToOne
    @JoinColumn(name="uuid_user")
    private NetflixUserEntity uuid_user;
```